demo/app/main-view-model.ts

```diff
@@ -48,27 +48,38 @@ export class HelloWorldModel extends Observable {
     };
 
     setTimeout(() => {
-      console.log('setupCallListener!');
+      console.log('Registering call listeners');
       setupCallListener(callListener);
-    }, 10000)
+    }, 15000)
 
-    // listener for push notifications (incoming calls)
-    const pushListener = {
-      onPushRegistered(accessToken, deviceToken) {
-        dialogs.alert('push registration succeded');
-      },
-      onPushRegisterFailure (error) {
-        dialogs.alert(`push registration failed: ${error}`);
-      }
-    };
+  
 
-    setupPushListener(pushListener);
+    setTimeout(() => {
+        // listener for push notifications (incoming calls)
+      const pushListener = {
+        onPushRegistered(accessToken, deviceToken) {
+          dialogs.alert('push registration succeded');
+        },
+        onPushRegisterFailure (error) {
+          dialogs.alert(`push registration failed: ${error}`);
+        },
+
+        onIncomingCall(customParameters) {
+          return {
+            from: customParameters.from_name
+          }
+        }
+      };
+      console.log('Registering push listeners');
+      setupPushListener(pushListener);
+    },20000);
   }
 
   public onCall(): void {
+    console.log('Updating access token:');
     getAccessToken()
       .then((token) => {
-        console.log(`Twilio access token: ${token}`);
+        // console.log(`Twilio access token: ${token}`);
```

Comment:
I think we would have this `console.log`. Why do you think we need the L79 one?

Reply:
I have removed the old comment and restored the twilio access token comment 💯